cmd/piper.go

```diff
@@ -265,14 +266,6 @@ func PrepareConfig(cmd *cobra.Command, metadata *config.StepData, stepName strin
 		GeneralConfig.NoTelemetry = true
 	}
 
-	if !GeneralConfig.Verbose && stepConfig.Config["verbose"] != nil {
```

Comment:
is this no more needed or how is this behavior mitigated?

Reply:
This just says dont take the value from the config when `GeneralConfig.Verbose` already says verbose is `true`
So we wanted to stop the config from disabling the verbose logging when it was enabled per command line argument. But this can be simplified by ignoring a `false` value from the config, since it's false per default.
If verbose logging is enabled via command line argument it has already been done at this point of the code since we have this line in every `_generated.go` file.

https://github.com/SAP/jenkins-library/blob/81c8553d6a2aa5119a02bfd3689f439fa75b14a6/cmd/sonarExecuteScan_generated.go#L95

So the behaviour we have we have after this PR is merged would be this

| GeneralConfig.Verbose | Config.yml | Verbose enabled |
|-----------------------|------------|-----------------|
| True                  | False      | True            |
| False                 | True       | True            |
| False                 | False      | False           |
| True                  | True       | True            | 

which is exactly the same as before.

Or maybe I am missing something obvious 😆

Reply:
I guess this makes sense :-)